polyfills/Intl/update.task.js

```diff
@@ -1,32 +1,43 @@
 'use strict';
```

Comment:
I know it wasn't there before, but a brief header comment describing what the postinstall task is needed for would add some useful context.